lib/install.js

```diff
@@ -32,6 +37,15 @@ function install(opts, cb) {
                 return cb1();
             });
 
+            if (process.env.HOT_RELOAD) {
+                if (typeof process.env.HOT_RELOAD_BASEDIR !== 'undefined') {
```

Comment:
If there are a multiple options for the hot reload option we should just make `opts.hotReload` a top level option.